examples/platform_view/ios/Podfile.lock

```diff
@@ -1,23 +1,16 @@
 PODS:
   - Flutter (1.0.0)
```

Comment:
This seems to be the only Podfile.lock in the entire repo.  You can see how it changes.
I don't know why MaterialControls was in there, but it doesn't seem to be a real dependency and was automatically removed?